internal/pkg/utils/testhelper/env.go

```diff
@@ -22,7 +22,7 @@ type EnvProvider interface {
 // ReplaceEnvsString replaces ENVs in given string.
 func ReplaceEnvsString(str string, provider EnvProvider) string {
 	return regexp.
-		MustCompile(`%%[a-zA-Z0-9\-_]+%%`).
+		MustCompile(`%%[a-zA-Z0-9][a-zA-Z0-9\-_]*[a-zA-Z0-9]%%`).
```

Comment:
V jednom teste mame expected text `%Y-%m-%d:%T%Z` (nie su to wildcards).
Neviem presne, ako je mozne ze to nepadalo :) Ale uz to pada.
Tak som muse `%` escapovat na `%%`, tj. `%%Y-%%m-%%d:%%T%%Z`, co uz preslo.

Ale nastal dalsi problem, ze `%%Y-%%` mi zobralo ako ENV placeholder.
Tak som na tomto mieste v kode upravil, ze ENV placeholder musi zacinat aj koncit alpha-numerickym znakom.